packages/dnb-eufemia/src/components/info-card/InfoCard.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react'
+import classnames from 'classnames'
+
+// Components
+import Button from '../button/Button'
+import { createSpacingClasses } from '../space/SpacingHelper'
+import { createSkeletonClass } from '../skeleton/SkeletonHelper'
+
+// Icons
+import {
+  lightbulb as LightbulbIcon,
+  close as CloseIcon,
+} from '../../icons'
+// Shared
+import Context from '../../shared/Context'
+import { ISpacingProps, SkeletonTypes } from '../../shared/interfaces'
+import { extendPropsWithContext } from '../../shared/component-helper'
+import IconPrimary, { IconPrimaryIcon } from '../icon-primary/IconPrimary'
+import Img, { ImgProps } from '../../elements/Img'
+import { Div, H3, P } from '../../elements'
+import { ButtonProps } from 'packages/dnb-eufemia/build/components/button'
+
+export interface InfoCardProps {
+  /**
+   * Aligns the content to center, rather than left
+   * Default: false
+   */
+  center?: boolean
```

Comment:
```suggestion
  centered?: boolean
```

Just a small suggestion, as I see us using the word _**centered**_ a few other places:
- https://eufemia.dnb.no/uilib/components/form-row/properties
- https://eufemia.dnb.no/uilib/components/modal/properties
-

Reply:
Done